src/PostgREST/ApiRequest.hs

```diff
@@ -142,16 +142,8 @@ userApiRequest schema req reqBody
       ActionInvoke{isReadOnly=True} -> partition (liftM2 (||) (isEmbedPath . fst) (hasOperator . snd)) flts
       _ -> (flts, [])
   flts = [ (toS k, toS $ fromJust v) | (k,v) <- qParams, isJust v, k /= "select", not (endingIn ["order", "limit", "offset", "and", "or"] k) ]
-  hasOperator val = foldr ((||) . flip T.isPrefixOf val) False ((<> ".") <$> (M.keys operators ++ ["not", show Plain, show Phrase]))
-                    || hasLanguageFts val
+  hasOperator val = foldr ((||) . flip T.isPrefixOf val) False $ ((<> ".") <$> "not":M.keys operators) ++ ((<> "(") <$> M.keys ftsOperators)
```

Comment:
This code is kind of gnarly, would the [any](https://hackage.haskell.org/package/base-4.10.0.0/docs/Prelude.html#v:any) function help simplify it?

Reply:
Like how about this:

```haskell
hasOperator val =
  any (`T.isPrefixOf` val)
    $  map (`snoc` '.') ("not" : M.keys operators)
    ++ map (`snoc` '(') (M.keys ftsOperators)
```

Reply:
The `any` function is helpful, though the `snoc/map` instead of `<>/<$>` makes the function harder to read for me.